crates/sui-types/src/temporary_store.rs

```diff
@@ -1288,22 +1320,20 @@ impl<S: GetModule + ObjectStore + BackingPackageStore> TemporaryStore<S> {
         // storage rebate pool
         if total_input_rebate != gas_summary.storage_rebate + gas_summary.non_refundable_storage_fee
```

Comment:
```suggestion
        if total_input_rebate != gas_summary.storage_rebate + gas_summary.non_refundable_storage_fee + gas_summary.unmetered_storage_rebate
```

was the fix that @lxfind suggested to make this pass for the change epoch transaction (if we want to roll that in here)